buddyboss/buddyboss-platform
# #2582 - fixed video button display issue on message compose page base…

### All Submissions:






### Changes proposed in this Pull Request:
added condition so that it checks the video access control settings



Fixes #2582 

### Related BuddyBoss Platform Pro PR
https://github.com/buddyboss/buddyboss-platform-pro/pull/176

### How to test the changes in this Pull Request:

1. Go to backend, buddyboss > settings > media > Media access
2. Set upload videos option
3. check frontend in private message compose page

### Proof Screenshots or Video
https://www.loom.com/share/da38d72c9137468ab399acd81e905363



### Other information:
